exporter/loadbalancingexporter/README.md

```diff
@@ -0,0 +1,143 @@
+# Trace ID aware load-balancing exporter
+
+This is a load balancer that will consistently export spans belonging to the same trace to the same backend.
+
+It requires a source of backend information to be provided: static, with a fixed list of backends, or DNS, with a hostname that will resolve to all IP addresses to use. The DNS resolver will periodically check for updates.
+
+Note that only the Trace ID is used for the decision on which backend to use: the actual backend load isn't taken into consideration. Even though this load-balancer won't do round-robin balancing of the batches, the load distribution should be very similar among backends with a standard deviation under 5% at the current configuration.
+
+IMPORTANT: this exporter assumes that all spans from the batch belong to the same trace. You *should* therefore use a processor that either splits the incoming batches into multiple batches, one per trace, or a processor that groups spans by trace, like the `groupbytrace`.
```

Comment:
>  that either splits the incoming batches into multiple batches, one per trace

Does this exist?  might be better just to mention `groupbytrace` if it's the only existing option.

Reply:
It will, once #1235 is addressed :-) I think `groupbytrace` isn't suitable for all cases, although I would certainly recommend it for most cases that I can think of.